bot/controllers/viewPoints.js

```diff
@@ -0,0 +1,38 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+const {
+  Member,
+} = require('../../db/models');
+
+class ViewPointsController extends BaseController {
+  constructor(message) {
+    super(message);
+    const controller = this;
+    this.commands = [
+      new Command(
+        '!viewPoints',
+        '!viewPoints',
+        'Return Users Current Points',
+        'Receive a message with the users current points.',
```

Comment:
If you could, I would try and re-word this say something like "Receive a message with your current point total" or something like that. It might sound better since it would be targeting an individual user instead of a group of users. The same would go for line 15 as well.